agent/agent.py

```diff
@@ -106,8 +106,8 @@ def load(self, fp):
 
         else:  # Create new
             logger.info("Create new runner [%s]" % fp)
-            module_path = "hasalTask"
-            object_name = "HasalTask"
+            module_path = "iskakalunan"
+            object_name = "Iskakalunan"
```

Comment:
@ShakoHo I'll suggest we create a separate issue to track this one instead of having one PR solves all problems.